tests/test_arraycreation.py

```diff
@@ -546,7 +549,9 @@ def test_full_like(array, fill_value, dtype, order):
 
     a = numpy.array(array)
     ia = dpnp.array(array)
-    assert_array_equal(func(numpy, a), func(dpnp, ia))
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", numpy.ComplexWarning)
```

Comment:
What behavior would be changed if to use `@pytest.mark.usefixtures("suppress_complex_warning")` instead?

Reply:
I don't think the behavior will change. 
This is just one way of handling warnings as an option and I think using `pytest.mark.usefixtures ` looks better and cleaner